installer.sh

```diff
@@ -299,7 +299,7 @@ case ${ANSWER:0:1} in
       docker pull fkiecad/fact_extractor:latest
     fi
     if ! [[ -f "./external/extract.py" ]]; then
-      download_file "FACT-extract" "https://raw.githubusercontent.com/fkie-cad/fact_extractor/master/extract.py" "external/extract.py"
```

Comment:
I am wondering why the download_file should not work in this situation. I will to another installation check on this